utils.js

```diff
@@ -488,3 +488,31 @@ export const getOptionsFromHash = (hash, label = 'name') => {
 
   return results
 }
+
+export const getPrice = (() => {
+  function max(p) {
+    if (typeof p === 'string' || typeof p === 'number') {
+      return p
+    }
+    if (p?.max === p?.min) {
+      return ''
+    }
+    const price = `${p?.max + ' ' + (p?.currency || ' ')}`
+    return price
+  }
+  function min(p) {
+    if (typeof p === 'string' || typeof p === 'number') {
+      return p
+    }
+    if (p?.min === p?.max) {
+      const price = `${p?.min + ' ' + (p?.currency || ' ')}`
+      return price
+    }
+    const price = `${p?.min || p?.amount} \u2014`
```

Comment:
There should be no space before/after en-dash: https://www.thepunctuationguide.com/en-dash.html